index.js

```diff
@@ -45,8 +45,20 @@ FOR j := 0 to 3
 ENDFOR
 dst[MAX:256] := 0
 `;
-// console.dir(parser.parse(vpermilpd), {depth: null});
 
+const test = `
```

Comment:
Can you give this the instruction name (looks like `pshufb`) please?

Reply:
It is the `_mm_shuffle_pi8` macro, which is `pshufb`, indeed.